<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parking</class>
 <widget class="QDialog" name="Parking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <widget class="QWidget" name="stackedWidgetPage1">
    <widget class="QPushButton" name="ADD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">backgroun-color : transparent;
border-image: url(:/img/img/button-304224_960_720.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="DELETE">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">backgroun-color : transparent;
border-image: url(:/img/img/abort-146072_960_720.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="EDIT">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">backgroun-color : transparent;
border-image: url(:/img/img/edit-validated_40458.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="SHOW">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>200</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">backgroun-color : transparent;
border-image: url(:/img/img/show-non-hidden-views.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
image: url(:/img/img/menu_btn_off.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_11</zorder>
    <zorder>ADD</zorder>
    <zorder>DELETE</zorder>
    <zorder>EDIT</zorder>
    <zorder>SHOW</zorder>
   </widget>
   <widget class="QWidget" name="stackedWidgetPage2">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>couleur</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>matricule</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>marque</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius : 10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stackedWidgetPage3">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>matricule</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stackedWidgetPage4">
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouv Matricule</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Couleur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Marque</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stackedWidgetPage5">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>561</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>retour</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>420</y>
       <width>271</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Matricule</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLineEdit" name="lineEdit_9"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="stackedWidgetPage6">
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: transparent;
border-image: url(:/img/img/images.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>240</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: transparent;
border-image: url(:/img/img/2531390_image_1.jpg);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color: transparent;
border-image: url(:/img/img/no-photo-vehicle.jpg);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
